src/System.Private.Xml/src/System/Xml/Serialization/XmlSchemaImporter.cs

```diff
@@ -418,66 +408,7 @@ private TypeMapping ImportElementType(XmlSchemaElement element, string identifie
 
         private void RunSchemaExtensions(TypeMapping mapping, XmlQualifiedName qname, XmlSchemaType type, XmlSchemaObject context, TypeFlags flags)
         {
-            string typeName = null;
```

Comment:
>string [](start = 12, length = 6)

why are you removing this code, but keeping others (behind #if CODEDOM)
would it make sense to be consistent one way or another?